tt_metal/include/compute_kernel_api/eltwise_unary/round.h

```diff
@@ -0,0 +1,45 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+
+#include "compute_kernel_api/common_globals.h"
+#ifdef TRISC_MATH
+#include "llk_math_eltwise_unary_sfpu_round.h"
+#define MAIN math_main()
+#define MATH(x) x
+#else
+#define MATH(x)
+#endif
```

Comment:
I check that `TRISC_MATH` is undefined on my platform.  Does this mean that the kernel function is unused?  I also find it puzzling because it is also the case for `floor`, and `floor` passes the tests in tenstorrent/pytorch2.0_ttnn#198.